pkgs/tools/networking/opensm/default.nix

```diff
@@ -16,6 +17,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = "bash ./autogen.sh";
```

Comment:
Not sure if this might be a source of problems but this could be simpler:
```rust
preConfigure = "./autogen.sh"
```

Let it be run in the current build env (sandboxed) with the current shell.